src/olympia/editors/models.py

```diff
@@ -29,8 +29,8 @@
 
 class CannedResponse(ModelBase):
 
-    name = TranslatedField()
-    response = TranslatedField(short=False)
+    name = models.CharField(max_length=255)
+    response = models.TextField()
```

Comment:
Done.  Is removing those lines enough or is something defined in translations also?